packages/extension-chakra-storefront/src/components/header/index.jsx

```diff
@@ -208,7 +208,7 @@ const Header = ({
                             onClick={onMyAccountClick}
                             onMouseOver={isDesktop ? onAccountMenuOpen : noop}
                         >
-                            <AccountIcon boxSize={6} />
+                            <AccountIcon boxSize="6" />
```

Comment:
Better to use string instead of {} because the design token system is optimized to work with string values, ensuring more reliable processing by the styling engine.